api/core/tools/provider/builtin/gitlab/tools/gitlab_commits.yaml

```diff
@@ -3,23 +3,23 @@ identity:
   author: Leo.Wang
   label:
     en_US: Gitlab Commits
-    zh_Hans: Gitlab代码提交内容
+    zh_Hans: Gitlab提交内容查询
```

Comment:
There should be a space between Chinese words and English words.

Reply:
Okay I will make these change